lib/src/covid19client.dart

```diff
@@ -51,29 +51,27 @@ class Covid19Client {
   /// It returns a list of [Covid19CountryAllStatus].
```

Comment:
Could you please change [Covid19CountryAllStatus] with [Covid19CountryEx]?
Thank you!